lib/build/bundle.js

```diff
@@ -5,6 +5,7 @@ const Convert = require('./convert-source-map');
 const fs = require('../file-system');
 const SourceInclusion = require('./source-inclusion').SourceInclusion;
 const DependencyInclusion = require('./dependency-inclusion').DependencyInclusion;
+const copyProperty = 'copy';
```

Comment:
Does this need to be a constant? Is there something wrong with just using something like `x.copy` instead of `x[copyProperty]`?